data/src/main/kotlin/com/vultisig/wallet/data/api/EvmApi.kt

```diff
@@ -73,7 +81,7 @@ class EvmApiFactoryImp @Inject constructor(
 
             Chain.BscChain -> EvmApiImp(
                 httpClient,
-                "https://api.vultisig.com/bsc/"
+                "https://api.vultisig.com/bnb/"
```

Comment:
The endpoint `https://api.vultisig.com/bsc/` does not return any response, while `https://api.vultisig.com/bnb/` does